crates/commonware-node/src/consensus/application/executor.rs

```diff
@@ -267,27 +274,30 @@ where
         Ok(())
     }
 
-    #[instrument(parent = &cause, skip_all)]
+    #[instrument(parent = &cause, skip_all, err)]
```

Comment:
Remove. We are reporting the error already as part of `forward_finalized` and `canonicalize`